src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -40,31 +39,30 @@ public class CaseUpdatedAboutToSubmitHandler extends ResponseEventsAboutToSubmit
     private final AssociatedCaseLinkHelper associatedCaseLinkHelper;
     private final AirLookupService airLookupService;
     private final DwpAddressLookupService dwpAddressLookupService;
-    private IdamService idamService;
+    private final IdamService idamService;
     private final RefDataService refDataService;
-    private final boolean workAllocationFeature;
+    private final boolean caseAccessManagementFeature;
 
-    @Autowired
     CaseUpdatedAboutToSubmitHandler(RegionalProcessingCenterService regionalProcessingCenterService,
                                     AssociatedCaseLinkHelper associatedCaseLinkHelper,
                                     AirLookupService airLookupService,
                                     DwpAddressLookupService dwpAddressLookupService,
                                     IdamService idamService,
                                     RefDataService refDataService,
-                                    @Value("${feature.work-allocation.enabled}")  boolean workAllocationFeature) {
+                                    @Value("${feature.case-access-management.enabled}")  boolean caseAccessManagementFeature) {
         this.regionalProcessingCenterService = regionalProcessingCenterService;
         this.associatedCaseLinkHelper = associatedCaseLinkHelper;
         this.airLookupService = airLookupService;
         this.dwpAddressLookupService = dwpAddressLookupService;
         this.idamService = idamService;
         this.refDataService = refDataService;
-        this.workAllocationFeature = workAllocationFeature;
+        this.caseAccessManagementFeature = caseAccessManagementFeature;
     }
```

Comment:
FYI, I love this method of injection rather than @Autowire as it makes unit testing so much more slick